agir/front/components/app/routes.config.js

```diff
@@ -252,6 +252,7 @@ export const routeConfig = {
     hasLayout: true,
     layoutProps: {
       desktopOnlyFooter: false,
+      displayOnMobileApp: true,
```

Comment:
C'est mieux d'être un peu plus explicite sur le fait qu'il s'agit du footer :
```suggestion
      displayFooterOnMobileApp: true,
```